src/test/java/com/sevendragons/practice/leetcode/medium/courseschedule/Solution.java

```diff
@@ -0,0 +1,71 @@
+package com.sevendragons.practice.leetcode.medium.coursesschedule;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Stack;
+
+public class Solution {
+
+	public static int	NUM_COURSE;
```

Comment:
Is there a really good reason for this static variable? It would be better to remove this. As a side note, the extra spaces between `int` and `NUM_COURSE` also look odd.